### DISCUSSION TITLE
Current/Future Override Rates

### DISCUSSION THREAD

**Discussion by psfshr** [BottlecapDave/HomeAssistant-OctopusEnergy#900] [General]
Is there any way to get current/future override rates?
The integration can get previous, current, & next day rates for the tariff I'm on, and can get previous rates for the override tariff, but not current or next day rates for the override tariff?
Not a critical thing I know, but would be useful when trying to compare tariffs to see the difference between upcoming rates.
I suspect it's maybe not something Octopus publish though?

**Comment by BottlecapDave** [comment]
Hello. The integration tries to limit the information that it downloads. Because there are no sensors that require current or future override rates, they're not downloaded.
